src/controllers/controllerengine.h

```diff
@@ -107,6 +107,7 @@ class ControllerEngine : public QObject {
     Q_INVOKABLE void scratchDisable(int deck, bool ramp = true);
     Q_INVOKABLE bool isScratching(int deck);
     Q_INVOKABLE void softTakeover(QString group, QString name, bool set);
+    Q_INVOKABLE void softTakeoverResync(QString group, QString name);
```

Comment:
can you call it reset instead of resync?  It could be interpreted to mean that the softtakeover is ignored